### ISSUE TITLE
can't get the point in SIFT gradient computation.

### ISSUE THREAD

**Issue by shawnlee103** [kmyid/learn-orientation#4]
According to paper, "We therefore use a numerical approximation of the gradients: when we form the training data we also compute the descriptors for many possible orientations, every 5 degrees in our current implementation. We can then efficiently compute the derivatives in @g/@theta by numerical differentiation."
what dose "numerical differentiation" means in this step?
how to do this in code exactly if already get descriptor every 5 degree? 
Is it possible to implement in tensorflow?

**Comment by kmyi** [comment]
It just means simple numerical approximation of the derivative. You compute the descriptor every five degrees, and then compute the numerical derivative, as you do with image gradients. 

In the follow-up works, we've moved towards using learned descriptors, which no longer requires this.
